src/inputs/dropdown/query-search-dropdown/PQuerySearchDropdown.vue

```diff
@@ -211,6 +212,10 @@ export default defineComponent<QuerySearchDropdownProps>({
         width: auto;
         min-width: 7rem;
         height: 1.125rem;
+        .input-element {
+            width: auto;
```

Comment:
I know that `width: auto;` in `input` is not applicable. When I looked it up, it seems that it is because it is affected by the property called size. I think it's unnecessary css, so please check it :)
---
`input`에서 `width: auto;` 는 적용 안되는 걸로 알고 있어요. 찾아보니, size라는 속성에 영향을 받아서 그렇다고 하네요. 불필요한 css일 것 같아 확인부탁드려요 :)